precompile/params.go

```diff
@@ -40,4 +44,38 @@ var (
 		TxAllowListAddress,
 		FeeConfigManagerAddress,
 	}
+	reservedRanges = []AddressRange{
+		{
+			common.HexToAddress("0x0100000000000000000000000000000000000000"),
```

Comment:
can we also add an explanation for the `0x01` range to the comment above?

Reply:
Good call, just added a comment to explain they come from coreth.